pwa/src/apiService/resources/authentication.ts

```diff
@@ -24,8 +24,8 @@ export default class Authentication {
     const { id } = variables;
 
     const { data } = await Send(this._instance, "DELETE", `/admin/authentications/${id}`, undefined, {
-      loading: "Removing authentication...",
-      success: "Authentication successfully removed.",
+      loading: "Removing authentication provider...",
+      success: "Authentication Provider successfully removed.",
```

Comment:
We should pick either "Authentication Provider" or "... provider". 

I think the easiest fix would be to rewrite these to "... Provider", seeing that in most places you've used the uppercase variant.